crates/bevy_ecs/src/world/error.rs

```diff
@@ -15,6 +16,22 @@ use crate::{
 #[error("The schedule with the label {0:?} was not found.")]
 pub struct TryRunScheduleError(pub InternedScheduleLabel);
 
+/// The error type returned by [`World::try_despawn`] if the provided entity does not exist.
+///
+/// [`World::try_despawn`]: crate::world::World::try_despawn
+#[derive(Error, Debug, Clone, Copy)]
+#[error("Could not despawn the entity with ID {0} because it {1}")]
+pub struct TryDespawnError(pub Entity, pub EntityDoesNotExistDetails);
```

Comment:
Named structs instead of tuple structs please. Operations on tuple structs are much less clear, so we avoid them as a matter of style unless they're 1-element newtypes.